k8s/upgrade/src/bin/upgrade-job/helm/client.rs

```diff
@@ -260,3 +272,61 @@ impl HelmReleaseClient {
         .fail()
     }
 }
+
+/// Installs CRDs which are missing from the target helm chart cluster which are missing
+/// from the cluster.
+async fn install_missing_crds(crd_client: &Api<Crd>, crd_dir_path: PathBuf) -> Result<()> {
+    ensure!(
+        crd_dir_path.is_dir(),
+        InvalidHelmChartCrdDir { path: crd_dir_path }
+    );
+    // List the entries in the 'crds' directory.
+    let entries = fs::read_dir(crd_dir_path.as_path())
+        .context(ReadingDirectoryContents {
+            path: crd_dir_path.clone(),
+        })?
+        .map(|res| res.map(|e| e.path()))
+        .collect::<Result<Vec<_>, std::io::Error>>()
+        .context(CollectDirEntries { path: crd_dir_path })?;
+
+    // Walk through the entries in the directory and create the CRDs.
+    // This errors out if a file is not a CRD, but that is okay because the 'crds' directory
+    // is meant for use with CRDs only.
+    for entry in entries {
+        if entry.is_file() {
+            let crd_yaml = fs::read(entry.as_path()).context(ReadingFile {
+                filepath: entry.clone(),
+            })?;
+
+            let crd: Crd = serde_yaml::from_slice(crd_yaml.as_slice())
+                .context(YamlParseFromFile { filepath: entry })?;
+
+            // Create CRDs, and ignore creation failures due to the CRD already
+            // existing in the cluster.
+            let _pp = PostParams::default();
```

Comment:
we use it right below..
```suggestion
            let pp = PostParams::default();
```